packages/inter-protocol/src/vaultFactory/params.js

```diff
@@ -139,7 +139,7 @@ harden(makeVaultDirectorParamManager);
  *   priceAuthority: ERef<PriceAuthority>,
  *   timer: ERef<import('@agoric/time/src/types').TimerService>,
  *   reservePublicFacet: AssetReservePublicFacet,
- *   loanTiming: LoanTiming,
+ *   interestTiming: InterestTiming,
```

Comment:
That's a breaking change to the RPC API, isn't it?

Seems like it would show up a [chainStorage snapshot test](https://github.com/Agoric/agoric-sdk/issues/6111#issuecomment-1481366083) if we had one for this part of `published`.

Reply:
Hmm, I'll look into this. Meanwhile I'll take the commit out of this PR.